_docs-v3/event-dragging-resizing/eventResize.md

```diff
@@ -11,7 +11,7 @@ function( *event*, *delta*, *revertFunc*, *jsEvent*, *ui*, *view* ) { }
 
 `event` is an [Event Object](event-object) that hold the event's information (date, title, etc).
 
-`delta` is a [Duration Object](moment-duration) that represents the amount of time the event's end was extended by. *Available in version 2.0.1 and later.*
+`delta` is a [Duration Object](moment-duration) that represents the amount of time (either start or end) was extended by or reduced by.
```

Comment:
doesn't grammatically make sense. change to "that represents the amount of time the event's start or end was extended or reduced by"